Sources/Sentry/PrivateSentrySDKOnly.m

```diff
@@ -134,4 +138,14 @@ + (NSData *)captureViewHierarchy
 
 #endif
 
++ (SentryUser *_Nullable)userWithJSONObject:(NSDictionary *)jsonObject
+{
+    return [[SentryUser alloc] initWithJSONObject:jsonObject];
+}
+
++ (SentryBreadcrumb *_Nullable)breadcrumbWithJSONObject:(NSDictionary *)jsonObject
+{
+    return [[SentryBreadcrumb alloc] initWithJSONObject:jsonObject];
+}
```

Comment:
m: To keep the pattern I suggest to ditch the "JSONObject" and use Dictionary.
And same for the actual implementation.